pom.xml

```diff
@@ -12,7 +12,7 @@
 
 	<artifactId>uniprot-rest-api</artifactId>
 	<groupId>org.uniprot</groupId>
-	<version>1.0.26-SNAPSHOT</version>
```

Comment:
we can declare the version as a property in the main pom (here) and reuse it elsewhere.